packages/a/ag-grid.json

```
@@ -16,24 +15,22 @@
     "grid",
     "table",
     "react",
-    "table",
     "angular",
     "angular-component",
-    "react",
     "react-component",
     "angularjs",
     "reactjs"
   ],
   "license": "MIT",
   "homepage": "http://www.ag-grid.com/",
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/ag-grid/ag-grid.git",
+    "source": "npm",
+    "target": "ag-grid-community",
     "fileMap": [
       {
-        "basePath": "grid-packages/ag-grid-community/dist",
+        "basePath": "dist",
         "files": [
-          "**/*.+(js|css)"
+          "ag-grid-community.min.js"
```

```suggestion
          "*.@(js|css)"
```
🤔 Do we not want all the css/js files still?